frontend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:18
+
+WORKDIR /app
+COPY . .
+RUN npm clean-install && npm run build && npm install http-server
+
+EXPOSE 5173
+
+CMD npx http-server dist --port 5173 --proxy $BACKEND_URL
```

Comment:
Because at the moment the frontend and backend containers are associated to different URLs in OpenShift, the frontend needs to know the URL for the backend to be able to communicate with it. I implemented this by using the BACKEND_URL environment variable in the frontend and set its value in the OpenShift UI.

Another option would be to use just one URL that is shared between the frontend and the backend e.g. by configuring OpenShift to route requests to `/api` path to the backend and everything else to the frontend.

Reply:
All right, thanks for clarification.